computer_vision_draft/houghTransform/houghTransform/main.cpp

```diff
@@ -0,0 +1,106 @@
+#include <opencv2/opencv.hpp>
+#include <iostream>
+#include <string>
+#include <fstream>
+#include<windows.h>  
+#include<chrono>
+#include<cmath>
+#include "DrivabilityDetector.h"
+
```

Comment:
Add a
```cpp
#define RACE
```
Here and comment it out for now

Reply:
Applied suggestion